releases/7.6.0.md

```diff
@@ -546,6 +538,8 @@ Minor incompatibilities:
     {% endblock arches_translations %}
     ```
 
+13. Run `python manage.py migrate`.
+
 14. Within your project, with your Python 3 virtual environment activated:
```

Comment:
Activating the environment should happen at least as early as migrate.

Reply:
Ah - good catch.